.github/workflows/deploy.yml

```diff
@@ -588,97 +588,32 @@ jobs:
           echo "IS_ALL_PLATFORMS_DEPLOYED=$isAllPlatformsDeployed" >> "$GITHUB_OUTPUT"
           echo "IS_ALL_PLATFORMS_DEPLOYED is $isAllPlatformsDeployed"
 
-  createPrerelease:
+  createRelease:
     runs-on: ubuntu-latest
-    if: ${{ always() && github.ref == 'refs/heads/staging' && fromJSON(needs.checkDeploymentSuccess.outputs.IS_AT_LEAST_ONE_PLATFORM_DEPLOYED) }}
+    if: ${{ always() && fromJSON(needs.checkDeploymentSuccess.outputs.IS_AT_LEAST_ONE_PLATFORM_DEPLOYED) }}
     needs: [prep, checkDeploymentSuccess]
     steps:
+      # v4.2.1
       - name: Download all workflow run artifacts
-        # v4
         uses: actions/download-artifact@95815c38cf2ff2164869cbab79da8d1f422bc89e
 
-      - name: 🚀 Create prerelease 🚀
+      - name:  🚀 Create release 🚀
         run: |
-          gh release create ${{ needs.prep.outputs.TAG }} --repo ${{ github.repository }} --title ${{ needs.prep.outputs.APP_VERSION }} --generate-notes --prerelease --verify-tag --target staging
+          gh release create ${{ needs.prep.outputs.TAG }} ${{ github.ref == 'refs/heads/staging' && '--prerelease' }} \
+            --repo ${{ github.repository }} \
+            --title ${{ needs.prep.outputs.TAG }} \
+            --generate-notes \
+            --verify-tag \
+            --target ${{ github.ref }}
           RETRIES=0
-          MAX_RETRIES=10
+          readonly MAX_RETRIES=10
```

Comment:
What's this do?

Reply:
huh TIL! https://unix.stackexchange.com/a/98402

Reply:
oh, it's just a good practice in bash. It is the same as `const` vs `let`. Only recently learned that this was a thing